marc_to_solr/lib/traject_config.rb

```diff
@@ -188,8 +190,19 @@
     accumulator << record.end_date_from_008
 end
 
-to_field 'cataloged_tdt', extract_marc('959a') do |record, accumulator|
-  accumulator[0] = Time.parse(accumulator[0]).utc.strftime("%Y-%m-%dT%H:%M:%SZ") unless accumulator[0].nil?
+# to_field 'cataloged_tdt', extract_marc('959a') do |record, accumulator|
+#   accumulator[0] = Time.parse(accumulator[0]).utc.strftime("%Y-%m-%dT%H:%M:%SZ") unless accumulator[0].nil?
+# end
```

Comment:
Keep this field the same please. The logic for checking for scsb makes this unnecessarily complex.